test/run-tests.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-cd "$( dirname "${BASH_SOURCE[0]}" )" && vim -Nu vimrc -c 'Vader! *' > /dev/null
+cd "$( dirname "${BASH_SOURCE[0]}" )" && /usr/local/bin/vim -Nu vimrc -c 'Vader! *' > /dev/null
```

Comment:
I'd rather not do this as it makes it harder to test things locally: you'd have to actually put vim under `/usr/local/bin` instead of using what is in your path. Can we move the absolute path to the `.yml` instead?